packages/web-components/src/components/ic-page-header/ic-page-header.e2e.ts

```diff
@@ -197,7 +197,7 @@ describe("ic-page-header component", () => {
       return topNav.offsetHeight;
     });
 
-    expect(pageHeaderTopValueOnMobile).toEqual(0 + topNavHeightOnMobile);
+    expect(pageHeaderTopValueOnMobile).toEqual(8 + topNavHeightOnMobile);
```

Comment:
The existing e2e test was failing and hadn't been picked up as e2e tests aren't in our static analysis, so this fixed it :)

Reply:
Just double-checked this test and I'm not sure if it's right. The `offsetTop` for the page header should be the height of the top navigation bar. The additional `8px` feels like it's come out of thin air, unless you know why? If not, it might be worth taking this test out so the test suite passes and then creating a ticket to refactor it.

Reply:
#365 ticked created for this :)